projects/components/src/modal/modal-container.component.scss

```diff
@@ -17,6 +17,11 @@
   }
 
   &.modal-size-medium {
+    height: 456px;
```

Comment:
so we relabeled medium to large and stuck in a new medium? be careful that any usages of medium get updated to large.